client/src/app/site/motions/services/motion-pdf-catalog.service.ts

```diff
@@ -116,13 +118,15 @@ export class MotionPdfCatalogService {
             text: this.translate.instant('Table of contents'),
             style: 'heading2'
         };
+        const exportSubmitterRecommendation = this.configService.instant<boolean>(
+            'motions_export_submitter_recommendation'
+        );
 
-        if (!sorting) {
-            sorting = this.configService.instant<string>('motions_export_category_sorting');
-        }
-        const exportCategory = sorting === 'identifier' || sorting === 'prefix';
+        // Initialize the header and the layout for border-style.
+        const header = exportSubmitterRecommendation ? this.getTocHeaderDefinition() : undefined;
+        const layout = exportSubmitterRecommendation ? BorderType.LIGHT_HORIZONTAL_LINES : BorderType.DEFAULT;
 
-        if (exportCategory && categories) {
```

Comment:
`exportCategory` was always false because sorting config name was wrong. So the toc was printed always _without_ categories. Now I removed the sorting part and pdfmake failed if categories are set and the config option to show submitters/recommendation is enabled.